.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Pipeline
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5, 2.6]
```

Comment:
I'm a little lost as to why we are running the job for ruby version 2.5 and 2.6?

Reply:
This was just to test against multiple versions of ruby. I'm fine with your suggestion and testing against 2.6.5 as we did with Jenkins

Reply:
Updated in [d115c98](https://github.com/Invoca/exceptional_synchrony/pull/77/commits/d115c98396e64dec26bec087ab9e659c2ef5d16e)

Reply:
nice!